boxes/react/package.json

```diff
@@ -46,7 +48,7 @@
   },
   "devDependencies": {
     "@types/jest": "^29.5.0",
-    "@types/node": "^20.5.9",
+    "@types/node": "^20.11.21",
```

Comment:
Curious why this was needed

Reply:
Most likely included by mistake, will roll back :)